.devcontainer/base-image/Dockerfile

```diff
@@ -12,6 +12,8 @@ RUN apt-get update \
 COPY Taskfile.yml /tmp/Taskfile.yml
 USER vscode
 
-RUN task -t /tmp/Taskfile.yml install-cargo-tools-binstall
+RUN task -t /tmp/Taskfile.yml install-cargo-tools-binstall \
+    && task install-pre-commit \
+    && pre-commit install-hooks
```

Comment:
Does this work?

Reply:
`pre-commit install-hooks` would make the commit fail when committing outside the container.

Reply:
> `pre-commit install-hooks` would make the commit fail when committing outside the container.

Hmm, why?

NB: this is different from `pre-commit install` — it just installs the current hooks, it doesn't have them run on each commit...

Reply:
> NB: this is different from `pre-commit install` — it just installs the current hooks, it doesn't have them run on each commit...

Ah, my mistake. Thanks for pointing that out.

Reply:
As an addendum, I think `task install-pre-commit` will fail because it cannot find the Taskfile.